.github/ISSUE_TEMPLATE/1.bug.yml

```
@@ -0,0 +1,55 @@
+name: Bug report 🐛
+description: Create a bug report for MUI Design kits.
+labels: ['status: needs triage']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide a searchable summary of the issue in the title above ⬆️.
+
+        Thanks for contributing by creating an issue! ❤️
+  - type: checkboxes
+    attributes:
+      label: Duplicates
+      description: Please [search the history](https://github.com/mui/mui-design-kits/issues) to see if an issue already exists for the same problem.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Latest version
+      description: We roll bug fixes and other improvements into new releases.
+      options:
+        - label: I have tested the latest version
+          required: true
+  - type: textarea
+    attributes:
+      label: Current behavior 😯
+      description: Describe what happens instead of the expected behavior.
+  - type: textarea
+    attributes:
+      label: Expected behavior 🤔
+      description: Describe what should happen.
+  - type: textarea
+    attributes:
+      label: Steps to reproduce 🕹
+      description: |
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug.
+
+        Issues that we can't reproduce will be closed.
+      value: |
+        Link to live example:
+
+        Steps:
+        1.
+        2.
+        3.
+        4.
+  - type: textarea
+    attributes:
+      label: Context 🔦
+      description: What are you trying to accomplish? How has this issue affected you? Providing context helps us come up with a solution that is more useful in the real world.
+  - type: textarea
+    attributes:
+      label: Your environment 🌎
+      description: The version of the design file you are using. The version is usually disabled in the footer of the design frames.
```

```suggestion
      description: The version of the design file you are using and, if it's Figma, whether you're using the browser or desktop web version.
```

A highlight around Figma's desktop and browser version. Also, got a bit confused by the last sentences, thus why I suggested removing it but happy to hear what you thought about it.